PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,16 +14,13 @@
 <!-- Briefly describe or list your changes  -->
 
 ## QA Notes
+Please make verification statements inspired by your code and what your code touches. 
```

Comment:
Could you remove the space after the period here?

Reply:
done!